template/scripts/run-tests

```diff
@@ -209,7 +217,22 @@ def release_file(
                             patch_version = version
                             ops_copy.remove((op, version))  # found an operator to patch
                             break
-                patched_release.append(line)
+                patched_release.append(line.rstrip("\n"))
+
+        # Filter out skip operators
+        skipped_release = []
+        for line in patched_release:
+            skip = False
+            if skip:
+                skip = False
+            else:
+                if any((op in line for op in skip_ops)):
+                    skip = True
+                    break
+                else:
+                    skipped_release.append(line)
+        patched_release = skipped_release
```

Comment:
I would have assumed something like `patched_release = "\n".join([line for line in patched_release if not any((op in line for op in skip_ops))])`. That said, this kind of string mangling of the YAML seems really fragile. Why not use a proper YAML parser instead?

Reply:
Your suggestion won't work because it needs to skip more than just one line.

> Why not use a proper YAML parser instead?

Seemed overkill back then. Maybe it's more appropriate now.

Reply:
> I'm confused, the PR seems to abort as soon as it encounters any skipped op?

You are absolutely correct. You pointed out to a bug that was also affecting the `--operator` argument. I fixed them both now.

Also I would love to use a yaml parser now , but before we do that, I want to move this script to a proper installable package.

Until then, please bare with me.

Reply:
Alright, that's fair.